src/styles/fonts.scss

```diff
@@ -0,0 +1,13 @@
+@font-face {
+  font-family: 'Inter';
+  src: url('../assets/fonts/Inter/Inter-Italic-VariableFont_opsz,wght.ttf') format('truetype');
+  font-weight: 100 900;
+  font-style: italic;
+}
```

Comment:
PDF generation library would not allow using a remote css file. We should have these downloaded anyway so we don't depend on a font server